simcore/simulation/library/crosslink.cpp

```diff
@@ -61,9 +62,12 @@ void Crosslink::SinglyKMC() {
   kmc_bind.SetPBCs(n_dim_, space_->n_periodic, space_->unit_cell);
 
   /* Calculate probability to bind */
-  double kmc_bind_prob = 0;
+  double kmc_bind_prob = 0;  
   std::vector<double> kmc_bind_factor(n_neighbors, k_on_d_);
   if (n_neighbors > 0) {
+    if (polar_affinity_ !=1){                                                
```

Comment:
Please use a linter/fixer to properly format whitespace surrounding operators for cleanliness and readability.